### DISCUSSION TITLE
Sonoff Zigbee USB Dongle Plus - error not connecting to device

### DISCUSSION THREAD

**Discussion by balajeek** [Koenkk/zigbee2mqtt#11905] [Devices]
I am currently using Conbee II with HA ZHA integration, wanting to change it to the better sonoff zigbee.
I ordered one that said CC2652P based usb 3.0 dongle.

I plugged in and followed the docker install method. put in this on docker-compose for the device:

`- '/dev/ttyUSB0:/dev/ttyUSB0'`
or
`- '/dev/serial/by-id/usb-ITead_Sonoff_Zigbee_3.0_USB_Dongle_Plus_d2f336cc6a29ec11a1386f7840c9ce8d-if00-port0:/dev/ttyUSB0'`

and on configuration.yaml

```
serial:
  port: /dev/serial/by-id/usb-ITead_Sonoff_Zigbee_3.0_USB_Dongle_Plus_d2f336cc6a29ec11a1386f7840c9ce8d-if00-port0
```

No matter i I put on the devices, it always come with error message:
```

Zigbee2MQTT:info  2022-03-19 19:04:50: Logging to console and directory: '/app/data/log/2022-03-19.19-04-49' filename: log.txt
Zigbee2MQTT:info  2022-03-19 19:04:50: Starting Zigbee2MQTT version 1.24.0 (commit #7a2ddf2)
Zigbee2MQTT:info  2022-03-19 19:04:50: Starting zigbee-herdsman (0.14.20)
Zigbee2MQTT:error 2022-03-19 19:04:50: Error while starting zigbee-herdsman
Zigbee2MQTT:error 2022-03-19 19:04:50: Failed to start zigbee
Zigbee2MQTT:error 2022-03-19 19:04:50: Check https://www.zigbee2mqtt.io/guide/installation/20_zigbee2mqtt-fails-to-start.html for possible solutions
Zigbee2MQTT:error 2022-03-19 19:04:50: Exiting...
Zigbee2MQTT:error 2022-03-19 19:04:50: Error: Error while opening serialport 'Error: Error: No such file or directory, cannot open /dev/ttyACM0'
    at SerialPort.<anonymous> (/app/node_modules/zigbee-herdsman/src/adapter/z-stack/znp/znp.ts:146:28)
    at SerialPort._error (/app/node_modules/zigbee-herdsman/node_modules/@serialport/stream/lib/index.js:198:14)
    at /app/node_modules/zigbee-herdsman/node_modules/@serialport/stream/lib/index.js:242:12
```

Why is it looking for /dev/ttyACM0, i have conbee II on it.  
So i unplugged conbee and tried again and its still the same.

Any help would be appretiated.

**Comment by balajeek** [comment]
Update:

Sorry i made a mistake, i created a folder zigbee2MQTT and in docker-compose i named id zigbee2mqtt, all along i was looking at the folder i created and did not realize i need to update the configuration.yaml on the zigbee2mqtt directory, I did that and it seem to load and shows in log "Started frontend on port 0.0.0.0:8083"  (yes, i changed the port in docker to 8083:8080 and "frontend port:8083" in configuration.yaml.

no errors, but i can't hit the frontend with http://localhost:8083 though. wondering what the problem is!

**Comment by balajeek** [comment]
another update: (just in case of someone in same boat trying to solve)

on the configuration.yaml the frontend port should be 8080 only, and change the port only on docker-compose file, where in my case i used port 8083:8080

I am able to load the frontend now and all seem to work.  I will starting pairing devices now.

**Comment by freaked1234** [comment]
Hey @balajeek , since I am doing the same swap (conbee2 -> Sonoff) too the next few days and I have over 40 devices paired. Did you have to re-pair everything? If not, could you describe how you did it?

**Reply by balajeek** [reply]
I don't think you can move paired devices from one brand controller to another.  So I kept the conbee2 setup until i moved all the devices to sonoff, it took time and i did not do it as one task, first i unpaired couple of devices from conbee and paired with sonoff, ofcourse the entity name changes so i go update HA UIs and automations accordingly, waited for few days to see how it plays out, then do few more on another day.  Luckly i do not have 40 devices, I am primarley on z-wave and just around 10 devices on zigbee.

one thing i could've tried is before removing the device from conbee, noted down the entity name, so after pairing with sonoff you try to rename the entity to the same name and hopefully you don't have to change HA UI cards or automations.

**Reply by alex-v-fraser** [reply]
@balajeek I moved from Raspbee II to Sonoff Zigbee 3 USB dongle with changing serial port from ‘/dev/ttyAMA0’ to ‘/dev/ttyUSB0’ without repeating pair devices.

**Reply by alex-v-fraser** [reply]
> Hey @balajeek , since I am doing the same swap (conbee2 -> Sonoff) too the next few days and I have over 40 devices paired. Did you have to re-pair everything? If not, could you describe how you did it?

@freaked1234 What’s your LQI-levels before and after migrating to Sonoff?
